### DISCUSSION TITLE
Per Page buttons - Holograms

### DISCUSSION THREAD

**Discussion by HeartlessAUS** [Zrips/CMI#3385] [Feature Requests]
**Description of feature/improvement request:**
Have an option for command/interaction per page, meaning each page can link to different sites, or do different commands
---
### Detailed explanation of what it should do, when and in what situation: ###
if the user has multiple pages, add an option for each page, possibly
in a wider sense, this could possibly be used for more then just per page commands, it can be used for all settings, but for what i need, its only commands

### Basic explanation as to why this feature is needed and/or useful to have: ###
it is useful for command pages, donation links on each page, or staff discord links

**Comment by HeartlessAUS** [comment]
any update if this could be a possible feature?

**Comment by Zrips** [comment]
Not currently, simple feature like this would require some changes for base command handling and its saving and its editing. Even tho it looks to be simple to add, it requite quite a bit of work
